mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -1223,6 +1233,23 @@ resource "google_sql_database_instance" "instance" {
 }
 `
 
+var testGoogleSqlDatabaseInstance_update_mssql = `
+resource "google_sql_database_instance" "instance" {
+  name                = "%s"
+  database_version    = "SQLSERVER_2019_STANDARD"
+  root_password       = "%s"
+  deletion_protection = false
+  settings {
+    tier = "db-custom-1-3840"
+    collation = "Polish_CI_AS"
```

Comment:
This won't force recreation because it's using the same collation as the previous config, but will update another field to ensure collation isn't sent & rejected?

Reply:
yep!

made sure in local testing that update request was sent here